cubedash/_filters.py

```diff
@@ -11,7 +11,7 @@
 import flask
 import pytz
 from datacube.index.fields import Field
-from datacube.model import Dataset, DatasetType, Range
+from datacube.model import Dataset, Product, Range
```

Comment:
I obviously approve, but this is breaking the documentation build - the documentation build must be using an ancient version of core.